easybuild/easyconfigs/p/phASER/phASER-bec9446.eb

```diff
@@ -7,18 +7,28 @@ homepage = "/home/umcg-ndeklein/dREG-master"
 description = """Detection of Regulatory DNA Sequences using GRO-seq Data.
 """
 
-toolchain = {'name': 'dummy', 'version': 'dummy'}
+#
+# Don't use 'dummy' as toolchain version to make sure
```

Comment:
The difference between 'dummy' and '' as toolchain version is documented, but subtle and as many use the code of existing easyconfigs as example/template for new ones I've added this explicitly... hoping this will result in more awareness of the difference and correct configs ;)